ui/appui-react/src/appui-react/syncui/SyncUiEventDispatcher.ts

```diff
@@ -213,11 +219,15 @@ export class SyncUiEventDispatcher {
           // istanbul ignore next
           if (args.previous.onViewChanged && typeof args.previous.onViewChanged.removeListener === "function")  // not set during unit test
             args.previous.onViewChanged.removeListener(SyncUiEventDispatcher._dispatchViewChange);
+          if (args.previous.onFeatureOverridesChanged && typeof args.previous.onFeatureOverridesChanged.removeListener === "function")  // not set during unit test
```

Comment:
As the source line was not tested, please add similar `//istanbul ignore next` so the coverage remains